sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,119 @@ This product is fully supported and enabled by default as of {product-title} 4.1
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in the `virtiofsd` daemon or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow the `virtiofsd` daemon to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* When upgrading the {sandboxed-containers-operator}, you might encounter an issue where the Machine Config Operator (MCO) pod changes to a `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the pod displays `sandboxed-containers-operator-scc` instead of the default `hostmount-anyuid` value.
++
+If this happens, temporarily change the `seLinuxOptions` strategy in the `sandboxed-containers-operator-scc` SCC to the less restrictive `RunAsAny`, so that the admission process does not prefer it over the `hostmount-anyuid` SCC.
++
+. Change the `seLinuxOptions` strategy by running the following command:
++
+[source,terminal]
+----
+$ oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "RunAsAny"}}'
+----
+
+. Restart the MCO pod by running the following commands:
++
+[source,terminal]
+----
+$ oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=0
+----
++
+[source,terminal]
+----
+$ oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=1
+----
+
+. Revert the `seLinuxOptions` strategy of the `sandboxed-containers-operator-scc` to its original value of `MustRunAs` by running the following command:
++
+[source,terminal]
+----
+$ oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "MustRunAs"}}'
+----
+
+. Verify that the `hostmount-anyuid` SCC is applied to the MCO pod by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-machine-config-operator -l k8s-app=machine-config-operator -o yaml | grep scc
+openshift.io/scc: hostmount-anyuid
+----
+
++
+(link:https://issues.redhat.com/browse/KATA-1373[*KATA-1373*])
```

Comment:
Hmmm... [*KATA-1373*](https://issues.redhat.com/browse/KATA-1373) was fixed in 1.3. It shouldn't appear in the know issues anymore AFAICT.

Reply:
@cpmeadors had said to leave it in, just in case users encounter an issue during the upgrade, since upgrading the cluster before upgrading to 1.3 can still cause problems.

Reply:
Removed this item as discussed [here](https://coreos.slack.com/archives/G01H4GSHP1B/p1659015301949219).